features/admin_center_manage_content.feature

```diff
@@ -562,22 +562,22 @@ Feature: Manage Content
     And I follow "RSS" within the Admin Center content
     And I follow "Add RSS Feed"
     When I fill in the following:
-      | Name  | Recalls                                                                          |
-      | URL 1 | http://www.fda.gov/AboutFDA/ContactFDA/StayInformed/RSSFeeds/HealthFraud/rss.xml |
+      | Name  | Recalls                                                                                |
+      | URL 1 | https://www.fda.gov/about-fda/contact-fda/stay-informed/rss-feeds/health-fraud/rss.xml |
     And I choose "RSS"
     And I add the following RSS Feed URLs:
-      | url                                                                                |
-      | http://www.fda.gov/AboutFDA/ContactFDA/StayInformed/RSSFeeds/FoodAllergies/rss.xml |
+      | url                                                                                      |
+      | https://www.fda.gov/about-fda/contact-fda/stay-informed/rss-feeds/food-allergies/rss.xml |
     And I submit the form by pressing "Add"
     Then I should see "You have added Recalls to this site"
     When I follow "Edit"
     Then the "Name" field should contain "Recalls"
-    And the "URL 1" field should contain "http://www.fda.gov/AboutFDA/ContactFDA/StayInformed/RSSFeeds/FoodAllergies/rss.xml"
-    And the "URL 2" field should contain "http://www.fda.gov/AboutFDA/ContactFDA/StayInformed/RSSFeeds/HealthFraud/rss.xml"
+    And the "URL 1" field should contain "https://www.fda.gov/about-fda/contact-fda/stay-informed/rss-feeds/food-allergies/rss.xml"
```

Comment:
Note: the cucumber matchers require these to be double rather than single quotes.

Reply:
👍